core/src/main/kotlin/org/sertain/hardware/Joysticks.kt

```diff
@@ -12,3 +12,31 @@ public fun Joystick.whenActive(buttonNumber: Int, command: Command) {
 public fun Joystick.whileActive(buttonNumber: Int, command: Command) {
     JoystickButton(this, buttonNumber).whileActive(command.mirror)
 }
+
+public fun Joystick.whenActive(buttonNumber: Int, block: () -> Unit) {
```

Comment:
`block`? Is this convention or just something you came up with? I think it could be a bit more clear.

Reply:
Convention, I agree with Aidan.

Reply:
Nice, alright 👍